### ISSUE TITLE
is this the pytorch-version of gnn?

### ISSUE THREAD

**Issue by dongtuu** [yanx27/3dgnn-pytorch#1]
Hi, I cannot find other way to contact you so I put up this issue. I was about to write a gnn of pytorch version when I get this. In the code I find the class EnetGnn. I wonder is this class the complete pytorch version of gnn at https://github.com/xjqicuhk/3DGNN ? I would like to do something with gnn, and I will appreciate it if your code can help.

**Comment by dongtuu** [comment]
sorry, replace 'gnn' with '3dgnn'

**Comment by yanx27** [comment]
Yes, this is pytorch version https://github.com/xjqicuhk/3DGNN , if you have any question we can discuss together, I would like to do something with 3dgnn, too :)
